applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -680,6 +665,20 @@ int handle_at_gps(enum at_cmd_type cmd_type)
 				return err;
 			}
 
+			/* Disable scheduled downloads for periodical tracking */
+			if (interval >= 10) {
+				err = nrf_modem_gnss_use_case_set(
+						NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START |
+						NRF_MODEM_GNSS_USE_CASE_SCHED_DOWNLOAD_DISABLE);
```

Comment:
I don't think so. The retry interval or the timeout in our AT command is NOT for scheduled download. To my understanding, schedule download is for long periodical mode. Before GNSS is re-started based on fix interval, it could stealthily start to receive from GPS in advance. Correct me if I am wrong, @tokangas

Reply:
Please see the log of `AT#XGPS=1,0.64,61` that demos the meaning of fix interval and fix retry (timeout):
00> [00:00:11.331,420] <inf> slm_gnss: GNSS_EVT_AGPS_REQ
00> [00:00:11.331,512] <inf> slm_gnss: GNSS started (cloud_assistance=0).
00> [00:01:12.336,090] <inf> slm_gnss: GNSS_EVT_SLEEP_AFTER_TIMEOUT
00> [00:01:15.342,041] <inf> slm_gnss: GNSS_EVT_PERIODIC_WAKEUP
00> [00:02:16.346,374] <inf> slm_gnss: GNSS_EVT_SLEEP_AFTER_TIMEOUT
00> [00:02:19.344,329] <inf> slm_gnss: GNSS_EVT_PERIODIC_WAKEUP
00> [00:03:20.348,785] <inf> slm_gnss: GNSS_EVT_SLEEP_AFTER_TIMEOUT
00> [00:03:23.346,801] <inf> slm_gnss: GNSS_EVT_PERIODIC_WAKEUP
00> [00:04:24.350,769] <inf> slm_gnss: GNSS_EVT_SLEEP_AFTER_TIMEOUT
00> [00:04:27.349,700] <inf> slm_gnss: GNSS_EVT_PERIODIC_WAKEUP

If there is no fix, the fix retyr will be at least 60s, so I used 61s.
Then if the fix interval is shorter than 60s, there will be no GNSS_EVT_PERIODIC_WAKEUP as GNSS runs continuously, so I used 64s.

Reply:
GNSS will do scheduled downloads for almanacs regardless of the set interval. For ephemerides, scheduled downloads are only done if interval is <= 30 minutes.

It takes some time before scheduled downloads start, because almanacs downloads need to be synced with the satellite broadcast. And the downloads can not start before GNSS gets the first fix.

Reply:
Removed 👍